backend/lib/scraper.js

```diff
@@ -383,13 +383,13 @@ function scrape_schedule(date_var){
                 case date_var < 20190323:
                   round = '1';
                   break;
-                case date_var >= 20190323 && date_var < 20190324:
+                case date_var >= 20190323 && date_var <= 20190324:
```

Comment:
I would suggest running all these date checks against a newly instantiated date, if possible. Similar to `var today = new Date();`

Reply:
Added to Project backlog.

Reply:
🎈